pkg/rekor/identity.go

```diff
@@ -403,11 +403,13 @@ func IdentitySearch(startIndex int, endIndex int, rekorClient *client.Rekor, mon
 		}
 		err = file.WriteIdentityMetadata(*idMetadataFile, idMetadata)
 		if err != nil {
-			return fmt.Errorf("failed to write id metadata: %v", err)
+			return nil, fmt.Errorf("failed to write id metadata: %v", err)
 		}
 	}
 
-	return nil
+	identities := identity.CreateIdentitiesList(monitoredValues)
```

Comment:
What is the reason for this change? How will the identity strings be used?

Reply:
the output of CreateIdentitiesList is used in [CreateMonitoredIdentities](https://github.com/sigstore/rekor-monitor/blob/b3b5cb9ee132ff15f784b1b4c00413766a3faaca/pkg/identity/identity.go#L99) to help create the structured output of found identity entries, which is passed in as the input for any attached notification platforms to create notifications off of

Reply:
Ah thanks, forgot we are grouping found entries by id. Looks good!